src/stac_iitmandi/templates/stac_iitmandi/base.html

```diff
@@ -1,17 +1,45 @@
+{% load static %}
+
 <!DOCTYPE html>
 <html lang="en">
 
 <head>
-  {% load static %}
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>{% block title %}STAC | IIT Mandi{% endblock %}</title>
+  <!-- Title of Page -->
+  {% if title %}
+    <title > STAC | IIT Mandi - {{title}}</title>
```

Comment:
I think it should be only `{{title}}` for better view.

Reply:
I think site name should be necessary with title.
@LalitNM  , What is your opinion on this?

Reply:
It should be like `<title of page> | STAC | IIT Mandi`, e.g. `Zenith | STAC | IIT Mandi`.

Reply:
We can probably do something like `{{title}} - STAC | IIT Mandi`.